RobotFramework_DoIP/DoipKeywords.py

```diff
@@ -0,0 +1,92 @@
+from robot.api.deco import keyword
+from robot.api import logger
+from robot.libraries.BuiltIn import BuiltIn
+
+from doipclient import DoIPClient
+
+class DoipKeywords(object):
+
+    def __init__(self):
+        address, announcement = DoIPClient.get_entity()
+        logical_address = announcement.logical_address
+        ip, port = address
+        print(ip, port, logical_address)
+
+        self.ip = ip
+        self.logical_address = logical_address
+        logger.info(f"IP: {ip} ")
+
+    @keyword
+    def connect(self):
+        pass
+    
+    @keyword
+    def send_diagnostic(self):
+        pass
+    
+    @keyword
+    def receive_diagnostic(self):
+        pass
+
+    @keyword
+    def reconnect(self):
```

Comment:
Hi Thong,

why did you  not implement this?
Doku says:
reconnect(close_delay=2)
Attempts to re-establish the connection. Useful after an ECU reset

Parameters:
close_delay (float, optional) – Time to wait between closing and re-opening socket

So, I expect the same interface as keyword.

Thank you,
 Thomas

Reply:
Hi Thomas,

I have update it as methods:
```python
 @keyword("Reconnect To Ecu")
    def reconnect_to_ecu(self, close_delay=A_PROCESSING_TIME):
```
Please take it for your info.

Thank you,
Thong Hua